### ISSUE TITLE
Add nice error handling to dialog flow 

### ISSUE THREAD

**Issue by seraphinatarrant** [seraphinatarrant/AlexaBots#13]
We need to go through the dialog flow and make the communication better - make sure error handling works out well and Alexa communicates when things go nicely or go wrong.

**Comment by cyoungs** [comment]
Agreed, I think MEC just quits if you do something "wrong"/  if she doesn't recognize your utterance as anything it knows what to do with. Unless this is another bug haha, then I'll file one for that.

<img width="1360" alt="screen shot 2018-05-10 at 3 37 31 pm" src="https://user-images.githubusercontent.com/22014571/39897676-293df0de-5468-11e8-8890-964a7427d2b7.png">
